server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -42,27 +42,32 @@ trap "rm -rf $tmp" EXIT
 mkdir -p "$tmp"
 
 log_init $(basename $0)
-logdir=$LOGSDIR/$(basename $0)/$prefix/$TS
-mkdir -p "$logdir"
-rc=$?
-if [[ $rc != 0 ]] ;then
-    echo Failed: mkdir -p "$logdir"
-    exit 2
-fi
+
+echo $TS: $PROG starting
 
 hosts="$(for host in $files;do echo ${host%%/*};done | sort -u )"
 tarballs=$(for tarball in $files; do echo ${tarball##*/}; done)
 
+typeset -i ntb=0
+typeset -i ntberrs=0
+typeset -i nstateerrs=0
+typeset -i nincomingerrs=0
+typeset -i nresultserrs=0
+
 for host in $hosts; do
     pushd $remotearchive/$host >/dev/null || exit 2
     for x in $tarballs; do
+        ntb=$ntb+1
         # remove tar file
         if [ -s $x ]; then
             rm $x
         fi
         rc=$?
         if [ $rc != 0 ]; then
-            echo Failed to remove: $x
+            echo Failed to remove: $remotearchive/$host/$x
+            ntberrs=$ntberrs+1
+            echo "Failed to remove the tarball from Archive" | mailx -s\
+            "$TS: $PROG: Failed to remove: $remotearchive/$host/$x" mail_recipients
```

Comment:
Accumulate all the errors from the run and send one mail out if there are any errors.